TheFourFifths/consus-client
# Make controller tff 116

### Summary

CONTROLLERS! 

All of the calls to the dispatcher, hash history, or the api have been moved to these controllers. 

### Checklist

- [x] Did you run `npm test`? ...You have no idea...

Finally, is it ready to be reviewed and merged: yes :dragon: 

### How to Review

1. Pull
2. npm install to get the new testing library.
3. npm test to ensure that all the tests work
4. Start up the server (dev will do)
5. Start up the client
6. Make sure everything works. I'm sorry if this is vague, but this PR has the potential to change anything.
7. Notice that code coverage is passing. 
8. Remember that API testing is another task so there's no tests on that yet.

### Links

Put links to Jira tickets, and anything else, here.

- [TFF-116](https://msoese.atlassian.net/browse/TFF-116)
